apps/wpcom-block-editor/src/calypso/features/iframe-bridge-server.js

```diff
@@ -1077,6 +1078,28 @@ async function preselectParentPage() {
 	}
 }
 
+function handleSiteEditorFeedbackPlugin( calypsoPort ) {
+	registerPlugin( 'a8c-fse-beta-feedback-plugin', {
+		render: () => {
+			const PluginSidebar = window?.wp?.editSite?.PluginSidebar;
```

Comment:
I assume this is in case the `edit-site` package doesn't have `PluginSidebar`, for example if it's an older version.

I've stumbled upon this in the same file:

https://github.com/Automattic/wp-calypso/blob/f31526a206fd0312b801e556191b5268fd67730a/apps/wpcom-block-editor/src/calypso/features/iframe-bridge-server.js#L22-L27

Would it make sense to reuse it here to decrease the optional chains, and maybe even to avoid registering the plugin altogether?
E.g.
```js
function handleSiteEditorFeedbackPlugin( calypsoPort ) {
  const PluginSidebar = editSitePackage?.PluginSidebar;

  if ( PluginSidebar ) {
    registerPlugin( /* ... */ );
  }
}
```

Reply:
> Would it make sense to reuse it here to decrease the optional chains, and maybe even to avoid registering the plugin altogether?

Maybe?  Probably!  I originally went with `window?.wp?.editSite?....` as I started in the ETK plugin which has some issues with using specific package versions.  Since the editSite package added to window.wp is generally the same version that the editor is running, using window.wp.editSite will generally be up to date while an import from an edit site dependency in ETK might not.  But now we aren't in ETK, so I think it should be fine to use the package.

Reply:
Updated!  That seems to work! 🥳   I was a bit worried as the last noted release of `@wordpress/edit-site` was about a month ago (before the PluginSidebar was added), but the PluginSidebar still seems to be available in the editSitePackage that is imported into `wpcom-block-editor`.  Perhaps the version listed in package.json isn't what actually ends up running and we get the same up to date version compiled from the gutenberg release we are running like in window.wp.editSite 🤷‍♀️